pages/common/multitail.md

```diff
@@ -0,0 +1,23 @@
+# multitail
+
+> Extension of tail.
```

Comment:
Is there a more information link we can include directly below this line? (example page that includes this syntax: `git`)

Reply:
I moved the link at the bottom. Is it ok to point directly to the examples sub-page?

Reply:
Fine by me!